README.md

```diff
@@ -506,6 +506,37 @@ Tags on keys in other languages will be ignored.
 [tags]: https://support.phrase.com/hc/en-us/articles/5822598372252-Tags-Strings-
 [configuration]: #Configuration
 
+#### Global key
+
+`vocab push` and `vocab pull` support global keys mapping.
```

Comment:
It may be worth to elaborate a little bit on what a global key is.
```suggestion
`vocab push` and `vocab pull` can support global keys mapping. When you want certain translations to use a specific/custom key in Phrase, add the `global-key` to the structure.
```